packages/hooks/src/__tests__/useSwitchToRealAccount.spec.tsx

```diff
@@ -1,71 +0,0 @@
-import * as React from 'react';
-import { mockStore, StoreProvider, useStore } from '@deriv/stores';
-import { renderHook } from '@testing-library/react-hooks';
-import useSwitchToRealAccount from '../useSwitchToRealAccount';
-import { createMemoryHistory } from 'history';
-import { Router } from 'react-router-dom';
```

Comment:
@aizad-deriv Please also remove `history` and `react-router-dom` from `packages/hooks/package.json` 🙏🏻

Reply:
Done 😩 👌🏼